src/content/en/2022/seo.md

```diff
@@ -1,22 +1,754 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: SEO
-#TODO - Review and update chapter description
 description: SEO chapter of the 2022 Web Almanac covering crawlability, indexability, page experience, on-page SEO, links, AMP, internationalization, and more.
 authors: [SophieBrannon, itamarblauer]
-reviewers: [mobeenali97, dwsmart, patrickstox, johnmurch, TusharPol]
-analysts: [derekperkins, csliva, jroakes]
+reviewers: [patrickstox, TusharPol, mobeenali97, dwsmart, johnmurch]
+analysts: [csliva, jroakes, derekperkins]
 editors: [MichaelLewittes]
 translators: []
 results: https://docs.google.com/spreadsheets/d/1qBQWxNKIAVJNOFwGlslT7AW0VAoK85Mf3nFvtw0QjVU/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
-unedited: true
+SophieBrannon_bio: TODO
```

Comment:
@SophieBrannon can you add a short bio here?

Reply:
@tunetheweb  

Sophie is the Client Services Director at UK-based agency Absolute Digital Media and specialises in SEO strategy and content marketing in highly competitive industries such as health and finance. Sophie is a conference speaker and industry blogger, and has proven experience in strategising and delivering award-winning campaigns on a local, national and international scale.

Reply:
Added